handlers.go

```diff
@@ -174,10 +168,37 @@ var BootstrapLoggingHandler = Handler{
 			m.logger.Debugf("setup logging: success")
 		}
 
+		if m.Cfg.FifoLogWriter != nil {
+			if err := captureFifoToFile(m.logger, m.Cfg.LogFifo, m.Cfg.FifoLogWriter); err != nil {
+				m.logger.Warnf("captureFifoToFile() returned %s. Continuing anyway.", err)
+			}
+		}
+
 		return nil
 	},
 }
 
+func createBootstrapLoggingHandler(logFifo string) Handler {
```

Comment:
Would it make sense to above this function do:
```
var BootstrapLoggingHandler = Handler{
  Name: BootstrapLoggingHandlerName,
  Fn: func(ctx context.Context, m *Machine) error {
    return createBootstrapLoggingHandler(m.Cfg.LogFifo)
  },
}
``` 

in order to dupe less code?

Reply:
Also changed